packages/uma-auth-client/src/components/UmaDisplay.tsx

```diff
@@ -0,0 +1,164 @@
+import { useTheme } from "@emotion/react";
+import styled from "@emotion/styled";
+import { Icon } from "@lightsparkdev/ui/components";
+import { Title } from "@lightsparkdev/ui/components/typography/Title";
+import {
+  motion,
+  useAnimationFrame,
+  useMotionTemplate,
+  useMotionValue,
+  useTransform,
+} from "framer-motion";
+import React, { useRef } from "react";
+
+const BORDER_RADIUS = "999px";
+
+export const UmaDisplay = ({
+  uma,
+  isLoading,
+}: {
+  uma?: string | undefined;
+  isLoading?: boolean;
+}) => {
+  const theme = useTheme();
+
+  return (
+    <Container>
+      <LoaderContainer>
+        <Loader>
+          <MovingBorder rx="30%" ry="30%">
+            {isLoading && <Gradient />}
+          </MovingBorder>
+        </Loader>
+        <UmaTagInnerBorder />
+        <UmaTagContainer>
+          <UmaTag>
+            <Title content={`$${uma}`} color={["content", "secondary"]} />
```

Comment:
Does the extra $ here mean that we intend to strip a leading $ from the actual value of `uma`?

Reply:
Currently the user inputs the value into the text input on the prior page using no $, so I was thinking of just storing the uma address without the $ sign so i don't have to remember to remove it. Could go either way though

Reply:
hmm... I think we'll want to be explicit about including the $ when we store this client side. For raw nwc connection uris, they can include the lud16 param which may or may not include a $.

Reply:
ok i'll do this in a follow up PR